colophon.php

```diff
@@ -32,10 +32,11 @@ function team51_credits( $args = array() ) {
 			)
 		);
 
-		$credit_links = array();
+		$credit_links   = array();
+		$parsed_url     = wp_parse_url( get_site_url(), PHP_URL_HOST );
+		$partner_domain = $parsed_url ? $parsed_url : 'wpspecialprojects.wordpress.com';
```

Comment:
Would it be better to have the fallback just to `wpspecialprojects.com` which is the mapped domain to that site?